src/lib/Tab/LocationView/ContentTab.php

```diff
@@ -60,12 +68,21 @@ public function renderView(array $parameters): string
         $contentType = $parameters['contentType'];
         $fieldDefinitions = $contentType->getFieldDefinitions();
         $fieldDefinitionsByGroup = $this->groupFieldDefinitions($fieldDefinitions);
+        $contentLanguages = $content->versionInfo->languageCodes;
+
+        $languages = $this->loadLanguages(
```

Comment:
You should probably also filter out disabled Languages.  Language can be disabled but it will be still visible in the content.